CMakeLists.txt

```diff
@@ -430,6 +430,13 @@ if (WIN32)
   add_compile_options($<$<CONFIG:Debug>:/homeparams>) # Force parameters passed in registers to be written to the stack
   endif (CLR_CMAKE_PLATFORM_ARCH_AMD64)
 
+  if(NOT CLR_CMAKE_PLATFORM_ARCH_ARM64)
```

Comment:
I am enabling it for all architectures on Windows aside from Arm64. Hence, the !ARM64.